rumqttc/examples/async_manual_acks_v5.rs

```diff
@@ -77,7 +77,7 @@ async fn main() -> Result<(), Box<dyn Error>> {
 async fn requests(client: &AsyncClient) {
     for i in 1..=10 {
         client
-            .publish("hello/world", QoS::AtLeastOnce, false, vec![1; i])
```

Comment:
Split client APIs to `publish` and `publish_with_properties`, `subscribe`, `subscribe_with_properties` etc